### PR TITLE
cleanup and refactor

### PR DESCRIPTION
**Pull request by radek1st**

### CODE REVIEW

File: tests/javascript-cucumber/Features/profile_update.feature [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Feature: User Profile Update
+
+  Scenario: User updates password
+
+    Given user logs in to their validated account
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/profile_update.feature:5]: @Soujanya2107 We should strive for simple and consice steps like this one

File: tests/javascript-cucumber/Features/signup.feature [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 
 Scenario: Create an user account
     Given user navigates to Relaypay personal signup page
     And fills in correctly all the required fields
     And they agree to terms and conditions
     When they click on the SIGN UP button
-    Then they see Email confirmation link sent response
+    And they see Email confirmation link sent response
+    And logging in with unverified email gives error
+    And confirm my account email is received and validated
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/signup.feature:12]: @addissemagn email api working beautifully!

File: tests/javascript-cucumber/Features/profile_update.feature (hunk 1 above)
Comment (Soujanya2107) [reply tests/javascript-cucumber/Features/profile_update.feature:5]: Sure Radek would do that, Thanks

File: tests/javascript-cucumber/Features/step_definitions/steps.js [hunk 3]
```diff
@@ -1,380 +1,373 @@
...
+        let buttonele = this.driver.wait(until.elementLocated(By.id('btn_change_password')), 60 * 1000);
+        await this.driver.wait(until.elementIsVisible(buttonele), 60 * 1000);
+        buttonele.click();
+    });
+
+    this.Then(/^user is displayed with success message$/, async function () {
+        await this.driver.wait(until.elementLocated(By.className('ant-alert-message'), 60 * 1000))
+    });
+
+    let signup = async function(driver, prefix= "") {
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/step_definitions/steps.js:336]: I added signup in one step function

**Pull request merged**